Main.py

```diff
@@ -38,7 +38,7 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
     world = MultiWorld(args.multi)
 
     logger = logging.getLogger()
-    world.set_seed(seed, args.race, str(args.outputname if args.outputname else world.seed))
+    world.set_seed(seed, args.race, str(args.outputname) if args.outputname else None)
```

Comment:
with you on these changes, but I'd like an explanation for this one.

Reply:
To my understanding, `world.seed` is always `None` in that spot, and it seemed like naming the seed `"None"` was not the intention. By passing `None`, `MultiWorld.set_seed` performs `self.seed_name = name if name else str(self.seed)` after the `seed` attribute is filled, which appears to be the intention of the caller.

Reply:
I feel like it wasn't always None and now I'm suspicious of that changing - though it should be fine.